src/lib/authorizer.go

```diff
@@ -15,6 +15,5 @@
 package lib
 
 import "github.com/goharbor/harbor/src/common/http/modifier"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380